### ISSUE TITLE
training datasets

### ISSUE THREAD

**Issue by sensen110** [liuziwei7/voxel-flow#1]
Hello.
How to build the training dataset from UCF?

**Comment by sesse** [comment]
I have a fork at https://git.sesse.net/?p=voxel-flow which contains, among others, a script to extract usable frames. I doubt it's exactly the same as the paper used (I assume that was some sort of MapReduce), but it allowed me to at least train a network with semi-reasonable results, so it can't be all bad.

**Comment by black107** [comment]
@sesse Could you tell me the meaning of "${PREFIX}.d1.psnr" in https://git.sesse.net/?p=voxel-flow ? Thank you!

**Comment by sesse** [comment]
The PSNR files are used to filter away frames without obvious motion; the dataset contains a lot of files that seem to have been upconverted from different frame rates. Basically I delay the file one frame and use ffmpeg to ask for PSNR between the two sequences, and if it's > 30 dB at any given spot (indicating a very good match), I can't use the triplet. Similarly, I delay it one more frame and check the PSNR again.

**Comment by elementCarbon** [comment]
@sesse Thanks for your comment. Could you tell me how do you get your training set? I download UCF dataset and randomly extract three consistent frames. Is this a proper way to generate the triplet? Or I misunderstand the meaning? Cause the result from this model is nothing near the one in the paper. The PSNR is only about 22 dB in cross validation and even worse in test set.

**Comment by sesse** [comment]
The script is in my git repository, linked above.

**Comment by sesse** [comment]
I don't support this code (I don't use it anymore).

**Comment by liuanping** [comment]
thanks @sesse，i have tried your scripts and work well.

**Comment by samo313** [comment]
> thanks @sesse，i have tried your scripts and work well.

how should I use to extract-ucf101.sh file?

**Comment by fmbao** [comment]
> I don't support this code (I don't use it anymore).

Hi @sesse  ,How to use generated image  to synthesize as video?
